azurelinuxagent/ga/env.py

```diff
@@ -98,6 +98,7 @@ def monitor(self):
         If dhcp client process re-start has occurred, reset routes.
         Purge unnecessary files from disk cache.
         """
+        self.protocol_util = get_protocol_util()
```

Comment:
I wanted to remove this property altogether but didnt mainly because a lot of test cases are dependent on this. I can take this up as a separate PR once I get back from vacation